python/ray/tune/experiment.py

```diff
@@ -167,6 +167,13 @@ def __init__(self,
         stopping_criteria = {}
         if not stop:
             pass
+        elif isinstance(stop, list):
+            if any(not isinstance(s, Stopper) for s in stop):
```

Comment:
be sure to document this later, and also note `list` as a possible return-type in the error message at end of the switch-case block.